src/test/groovy/uk/gov/justice/digital/hmpps/iis/EditSearchResultsSpec.groovy

```diff
@@ -0,0 +1,87 @@
+package uk.gov.justice.digital.hmpps.iis
+
+import spock.lang.Stepwise
+import uk.gov.justice.digital.hmpps.iis.pages.EditDobPage
+import uk.gov.justice.digital.hmpps.iis.pages.EditNamesPage
+import uk.gov.justice.digital.hmpps.iis.pages.SearchPage
+import uk.gov.justice.digital.hmpps.iis.pages.SearchResultsPage
+
+@Stepwise
+class EditSearchResultsSpec extends SignOnBaseSpec {
+
+    def setupSpec() {
+        signIn()
+    }
+
+    def cleanupSpec() {
+        signOut()
+    }
+
+    def 'can edit a name search'() {
+
+        when: 'I perform an name search and click to edit'
+        performSearchAndClickEditFirstTerm(['names'], [surname: 'surnamea'])
+
+        then: 'I change surname'
```

Comment:
only use 'then' for an assertion. This should be 'and' because it's part of the 'when'